platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -344,6 +344,21 @@ const mergeProps = (propsFromState, propsFromDispatch, ownProps) => {
         },
       });
     },
+    saveToSR: () => {
+      const successHandler = () => {
+        alert('Success');
```

Comment:
@dannyrb We should consider adding some generic notification toast which can be called for stuff like this

Reply:
https://github.com/OHIF/Viewers/issues/985

Reply:
We should use a notification toast, but in order to use it here, in an extension, we need to come up with a better approach on how to expose the notification into external extensions. 
Danny and I are working on that in a separeted branch, once we finish it, I can update this.

Reply:
Definitely not a blocker for this PR

Reply:
I've changed the measurementTable component to start using Snackbar for saving feedbacks.